lib/wovnrb/html_replacers/link_replacer.rb

```diff
@@ -12,9 +19,18 @@ def replace(dom, lang)
         href = node.get_attribute('href')
         next if href =~ /^\s*\{\{.+\}\}\s*$/
         next if href =~ /javascript:void\(/
+        next if is_file?(href)
         new_href = lang.add_lang_code(href, @pattern, @headers)
         node.set_attribute('href', new_href)
       end
     end
+
+    def is_file?(href)
+      img_files = /^(https?:\/\/)?.*(\.(#{FileExtension::IMG_FILES}))$/i
```

Comment:
As @yeuem1vannam suggested, regexp should handle query and hash:
```
img_files = /^(https?:\/\/)?.*(\.(#{FileExtension::IMG_FILES}))(\?|#|$)/i
```

Reply:
Also please add tests accordingly

Reply:
Changed regex to include query and hash and added tests. Thank you!

Reply:
thanks